.github/workflows/deploy_branch.yml

```
@@ -13,29 +13,76 @@ jobs:
       with:
           creds: ${{ secrets.AZURE_STATIC_SITES }}
 
-    - name: Use Node.js
+    - name: Node.js build
+      id: build
       uses: actions/setup-node@v1
       with:
         node-version: '14.x'
     - run: npm install
     - run: npm run _build-production
 
     - name: Upload to blob storage
+      id: upload
       uses: azure/CLI@v1
       with:
         creds: ${{ secrets.AZURE_STATIC_SITES }}
         inlineScript: |
             az storage blob upload-batch \
               --account-name zooniversestatic \
-              --content-cache-control 'public, max-age=604800, immutable' \
+              --content-cache-control 'public, immutable, max-age=604800' \
               --destination '$web/preview.zooniverse.org/pfe-lab/pr-${{ github.event.number }}' \
               --source ./dist
             az storage blob upload \
               --account-name zooniversestatic \
-              --content-cache-control 'max-age=60' \
+              --content-cache-control 'public, immutable, max-age=60' \
```

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control#revalidation_and_reloading
> `immutable`
> Indicates that the response body will not change over time. 

Make sure we don't add `immutable` to index.html cache headers as this file will change over time
```suggestion
              --content-cache-control 'public, max-age=60' \
```